lib/valkyrie/persistence/fedora/persister/model_converter.rb

```diff
@@ -181,14 +181,76 @@ def result
         end
       end
 
+      class IntegerValue < ::Valkyrie::ValueMapper
+        FedoraValue.register(self)
+        def self.handles?(value)
+          value.is_a?(Property) && value.value.is_a?(Integer)
+        end
+
+        def result
+          # cast it to datetime for storage, to preserve miliseconds and date
```

Comment:
This comment seems strange for an integer.

Reply:
copy pasta